### ISSUE TITLE
Swift Package Manager - support for all suite

### ISSUE THREAD

**Issue by salvatoreboemia** [Masonry/Snap#802]
How we can migrate this pod to SPM

  pod 'SnapSDK' -> We have a URL link for SPM

But for these one we not have a link in order to migrate to SPM

  pod 'SCSDKCameraKit'
  pod 'SCSDKCameraKitReferenceSwiftUI'
  pod 'SCSDKCameraKitReferenceUI'

In the official docs:

> Swift Package Manager
> 
>     Drag and drop the CameraKit/CameraKitReferenceSwiftUI directory into your project or workspace.
>     Click the + button in the “Frameworks, Libraries, and Embedded Content” section, select SCSDKCameraKitReferenceSwiftUI, and add it as a dependency.
> 

What does mean "drag and drop". SPM URL is not available? 

Thanks

**Comment by salvatoreboemia** [comment]
Hi. News?

**Comment by wlxo0401** [comment]
@salvatoreboemia 
The library you mentioned seems to be from Snapchat, and it doesn't appear to be related to SnapKit. The names seem to be similar.

Please let me know if I'm mistaken.

**Comment by salvatoreboemia** [comment]
> @salvatoreboemia The library you mentioned seems to be from Snapchat, and it doesn't appear to be related to SnapKit. The names seem to be similar.
> 
> Please let me know if I'm mistaken.

Yes the other "packages" is from Snapchat officially. But for now only the main package is available as SPM, not all the suite

because in the snapkit SDK the above SDK mentioned are not included.
